strongbox-web-core/src/main/java/org/carlspring/strongbox/config/WebConfig.java

```diff
@@ -187,6 +195,22 @@ public Validator localValidatorFactoryBean()
     {
         return new LocalValidatorFactoryBean();
     }
+    
+    @Bean
+    @Qualifier("loggingManagementDirectoryListingService")
+    public DirectoryListingService getLoggingManagementDirectoryListingService()
+    {
+        String baseUrl = StringUtils.chomp(configurationManager.getConfiguration().getBaseUrl(), "/");
+        return new DirectoryListingServiceImpl(String.format("%s/api/logging", baseUrl));
+    }
+
+    @Bean
+    @Qualifier("browseRepositoryDirectoryListingService")
+    public DirectoryListingService getBrowseRepositoryDirectoryListingService()
+    {
+        String baseUrl = StringUtils.chomp(configurationManager.getConfiguration().getBaseUrl(), "/");
+        return new DirectoryListingServiceImpl(String.format("%s/api/browse", baseUrl));
```

Comment:
Could the String parameter be taken out to a new line, for better readability?
```suggestion
        String browseBaseUrl = String.format("%s/api/browse", baseUrl);
        return new DirectoryListingServiceImpl(browseBaseUrl);
```